### DISCUSSION TITLE
Some confusion about a paper in CMAME

### DISCUSSION THREAD

**Discussion by Dreamown** [neperfepx/neper#613] [Q&A] [answered]
I have some confusion about the article published in CMAME given on the neper website. This is the article Doi: https://doi.org/10.1016/j.cma.2017.10.029
 The confusion are as follows:
1. Is the role of the H function in equation (6) to fit the discrete data {xi} with a step function?
![方程6](https://user-images.githubusercontent.com/32787082/203988911-3dfdacde-04f7-477f-b3a2-b5b688872162.png)
2. In comparing X(x) with F(x), equation (7) was chosen; is this a standard data analysis step? Or did you come up with it yourself, which I think is ingenious.
![方程7](https://user-images.githubusercontent.com/32787082/203989995-f3bd1bb1-7762-4dbd-9183-2684368ed381.png)
3. How is the error function in the equation chosen? Is there a selection criterion? How is equation (11) derived from (6) and (9)?
![方程11](https://user-images.githubusercontent.com/32787082/203992082-c5b15086-c75b-4d23-aba6-116ac03e1999.png)
![方程6](https://user-images.githubusercontent.com/32787082/203992104-479adaac-57ad-4f4d-859f-986b70e0a8f7.png)
![方程9](https://user-images.githubusercontent.com/32787082/203992123-cee2183d-da67-4749-9d44-78ee09860f96.png)
4.  What kind of strategy should this be to ensure that the initial distribution of spheres achieves the smallest possible overlap, please? My idea is to double count the distance between the seeds and the surrounding seeds at the time of seed deployment to be greater than the sum of the radii. Is there a better way to do this?
![策略](https://user-images.githubusercontent.com/32787082/203992564-a7ab761f-dadd-4f02-9ef5-024c3006fde2.png)
Since I am a materials researcher and not good at knowledge in statistical mathematics, I hope you can help me or recommend some articles and books.

**Comment by rquey** [comment, marked as answer]
Sorry for the delayed response...

1. Yes, the H function is used to construct the cumulative probability distribution function (CDF) of the discrete data. For each data, the CDF is increased by 1/N.
2. This is similar to an Anderson-Darling test, see e.g. https://en.wikipedia.org/wiki/Anderson%E2%80%93Darling_test.
3. The error function comes from integrating the normal distribution (kernel) representing each discrete data.
4. Yes, this is how I did it (rough dense sphere packing algorithm), stating by the largest weights/seeds.
